src/cookbook/initializing.md

```diff
@@ -71,6 +73,21 @@ location:
     $ echo 'PYTHONPATH=$PYTHONPATH:/usr/local/samba/lib/python2.7/site-packages;export PYTHONPATH' | sudo tee -a /etc/profile.d/samba4-env-build.sh
     $ . /etc/profile.d/samba4-env-build.sh
 
+# Setting up OpenChange dependencies #
```

Comment:
@ydeverdiere, @jelmer they are ... but not with Ubunut trusty. Only with more recent such as vivid

I would therefore add a mention libmemcached 1.0.18 and nanomsg 0.4 are available in packages as suggested by @jelmer